spec/features/goal_feature_spec.rb

```diff
@@ -20,14 +20,47 @@
     end
   end
 
-  describe 'POST /goals' do 
-    it 'creates a goal' do
-      goal_params = {name: "Learn piano"}.to_json
+  describe 'POST /goals' do
+    context 'create goals' do
```

Comment:
I would remove context until we have more tests. (Right now we have a describe and context for just one test).